css/portal-dashboard/student-names.less

```diff
@@ -14,17 +14,31 @@
     &.compact {
       height: 32px;
     }
-
     .name {
       margin: 0 20px 0 20px;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
     }
+    .checkbox {
+      width: 20px;
+      height: 20px;
+      border-radius: 5px;
+      border: 1.5px solid @cc-charcoal-light1;
+      background-color: white;
+      margin-left: 25px;
+    }
 
   }
-  .studentName:nth-child(even) {
+  .studentNameRow:nth-child(even) {
```

Comment:
I think the changes in this module break the student name layout.  The alternating cell coloring is also broken.  Every other student name cell should be teal.  Now they are all white.  Are any of the changes in this module used?